test/Unipool.js

```diff
@@ -252,5 +256,39 @@ contract('Unipool', function ([_, wallet1, wallet2, wallet3, wallet4]) {
 
             expect(await this.unipool.rewardRate()).to.be.bignumber.lessThan(originalRewardRate);
         });
+
+        it('requires nonzero reward to reinvest', async function () {
```

Comment:
I was actually intending to test that they would revert when their balance was 0, but I'll make two separate tests because they're both important cases.